readme.md

```diff
@@ -53,7 +66,6 @@ healthcheck:
 ```
 
 Source [[1]](https://github.com/minio/minio/blob/c9d502e6fadf2c2cbb1e2ce7a9dc8de7dfed1d34/docs/orchestration/docker-swarm/docker-compose-secrets.yaml#L25-L29) [[2]](https://github.com/minio/minio/blob/bdd094bc39275a0543484c09f4fb99ce3fe14787/docs/metrics/healthcheck/README.md)
-
```

Comment:
The PR looks good overall. I will add back this empty line to adhere to `markdownlint` recommendations.